recipes/rb-wisper/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "wisper" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 6d8623c76943612e8db81862af7eb20f5a8ad631fe1857d9f7ced97e87e5222b
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install --norc -l -V --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
```

Comment:
You do not need to unpack if you ship the license, but I guess we leave this in and hope upstream will ship a license at some point?